nvflare/fuel/hci/server/sess.py

```diff
@@ -147,3 +154,21 @@ def handle_list_sessions(self, conn: Connection, args: List[str]):
                     "{}".format(time.time() - s.last_active_time),
                 ]
             )
+
```

Comment:
might want to change the "handle_list_sessions"'s docstring.

And not sure if list(self.session.values) should be inside the lock.

Reply:
I don't think python has any read locks unfortunately, but I think it should be fine just to use the same lock.